CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -448,54 +511,79 @@ private Point GetNewCursorPos()
             };
         }
 
+        /// <summary>
+        /// Enter the TexasHoldEm gameflow
+        /// </summary>
         private void Initialize()
         {
+
+            //creates user and number of ai opponents
+            _players.InitiatePlayers(5);
+
             _gameOver = false;
             _userPlaying = false;
             _roundFinished = false;
             _userFolded = false;
-            _mainPot = 0;
-            // _blindIncreaseCountdown = 5;
+
+            //set initial blinds and ante, set countdown to increase blinds after 4 rounds of play
+            _ante = 2;
```

Comment:
Could you add a comment being a little more descriptive as to why these values are being set this way? I assume its a texas hold em specific thing but it will make any bug fixing later on down the road WAAAY easier.